.github/workflows/integration-tests.yml

```
@@ -44,3 +44,22 @@ jobs:
       vm_type: ${{ matrix.vm_type }}
       collector-tag: ${{ inputs.collector-tag }}
     secrets: inherit
+
+  notify:
+    runs-on: ubuntu-latest
+    if: always() && contains(join(needs.*.result, ','), 'failure') # && github.event_name == 'push'
+    needs:
+      - required-integration-tests
+      - all-integration-tests
+    steps:
+      - name: Slack notification
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_ONCALL }}
+          SLACK_CHANNEL: oncall
```

Could we also move them to the `collector-notifications` channel? I don't really see the value in sending these to an oncall channel.
```suggestion
          SLACK_WEBHOOK: ${{ secrets.SLACK_COLLECTOR_NOTIFICATIONS_WEBHOOK }}
          SLACK_CHANNEL: collector-notifications
```

---

Sure, sounds good